src/components/home-page/FeaturedBrands.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import santaCruzImg from "../../assets/santa-cruz-brand.png";
+import urbanMonkeyImg from "../../assets/urban-money-brand.png";
+import vansImg from "../../assets/vans-brand.png";
+import thrasherImg from "../../assets/thrasher-brand.png";
```

Comment:
Suggestion : you can make barrel export for images where you export all the images in one file and import it from one file at here.

P.S: You can do barrel export for components ,pages, contexts etc..

Reply:
oh i didnt think of that, i will implement it in the next PR